grails-app/services/io/xh/toolbox/App/NewsService.groovy

```diff
@@ -0,0 +1,88 @@
+package io.xh.toolbox.App
+
+import io.xh.hoist.json.JSON
+import io.xh.toolbox.BaseService
+import io.xh.toolbox.NewsItem
+import org.grails.web.json.JSONArray
+import org.grails.web.json.JSONObject
+
+import static io.xh.hoist.util.DateTimeUtils.MINUTES
+
+class NewsService extends BaseService {
+
+    public List<NewsItem> _newsItems
+
+    static clearCachesConfigs = ['newsSources', 'newsApiKey']
+    def configService
+
+    void init() {
+        log.info('news service initialized')
+        createTimer(
+                runFn: this.&loadAllNews,
+                interval: 10 * MINUTES
+        )
+        super.init()
```

Comment:
The framework should already be logging this initialization.

If we were going to log this, we would log at *end* of method, and capitalize as, say. 'NewsService Initialized'.